library/src/main/java/com/bumptech/glide/load/resource/bitmap/TransformationUtils.java

```diff
@@ -74,9 +76,11 @@ public static Bitmap centerCrop(@NonNull BitmapPool pool, @NonNull Bitmap inBitm
     // We don't add or remove alpha, so keep the alpha setting of the Bitmap we were given.
     TransformationUtils.setAlpha(inBitmap, result);
 
-    Canvas canvas = new Canvas(result);
```

Comment:
Can you provide documentation? Or is this reproducible on a device or device(s) in a sample app?